ush/yaml_from_parm.sh

```diff
@@ -7,12 +7,12 @@ if [[ "$1" == "jedivar" ]]; then
       -e "s/@HYB_WGT_STATIC@/${HYB_WGT_STATIC}/" -e "s/@HYB_WGT_ENS@/${HYB_WGT_ENS}/" \
       "${EXPDIR}/config/jedivar.yaml" > jedivar.yaml
   if [[ "${HYB_WGT_ENS}" == "0" ]] || [[ "${HYB_WGT_ENS}" == "0.0" ]]; then # pure 3DVAR
-    sed -i '127,152d' ./jedivar.yaml
+    sed -i '/- covariance:/,/HYB_WGT_ENS/d' ./jedivar.yaml
   elif [[ "${HYB_WGT_STATIC}" == "0" ]] || [[ "${HYB_WGT_STATIC}" == "0.0" ]] ; then # pure 3DEnVar
-    sed -i '80,126d' ./jedivar.yaml
+    sed -i '/- covariance:/,/HYB_WGT_STATIC/d' ./jedivar.yaml
```

Comment:
For line 12:
`sed -i '/- covariance:/{N;/covariance model: SABER/{:a;N;/HYB_WGT_STATIC/!ba;d}}' jedivar.yaml`

Reply:
I did not test JEDIVAR. Should there be $ sign before HYB_WGT_STATIC and HYB_WGT_STATIC?